test/test_utils.py

```diff
@@ -168,6 +168,13 @@ def test_draw_invalid_boxes():
         utils.draw_bounding_boxes(img_correct, boxes, colors=colors_wrong)
 
 
+def test_draw_boxes_warning():
+    img = torch.full((3, 100, 100), 255, dtype=torch.uint8)
+
+    with pytest.warns(UserWarning, match="Argument 'font_size' will be unused as 'font' is not set."):
```

Comment:
The string passed to `match` is actually a regex pattern. Thus, if you have special regex characters like `.` in it, you should wrap it in `re.escape` to mark them as "regular". For example

```py
>>> import re
>>> match = "'font' is not set."
>>> re.escape(match)
"'font'\\ is\\ not\\ set\\."
```

```suggestion
    with pytest.warns(UserWarning, match=re.escape("Argument 'font_size' will be ignored since 'font' is not set.")):
```

Note that you will have to `import re` at the top of the module.

Reply:
Noted. Will add the change. 👍

Reply:
@lezwon You forgot to update the warning message to match the one that is emitted from the function.

Reply:
Sorry. Missed noticing this somehow. Have updated it now.